simple-singolo/style.css

```
@@ -0,0 +1,1017 @@
+  --slider-main-background: #f06c64;
+  --slider-main-shifted: #648bf0;
+  --lite-background: #f2f2f2;
+  --bprder-dark-color: #323746;
+  --bprder-slider-main-color: #ea676b;
+  --arrow-color: #e94348;
+  --shade-color: #cd5d56;
+}
+
+@keyframes pulse {
+  0% {
+    transform: translateX(0vw);
+  }
+  50% {
+    transform: translateX(50vw);
+  }
+  100% {
+    transform: translateX(100vw);
+  }
+}
+.element {
+  position: absolute;
+  top: 0px;
+  left: -100vw;
+  z-index: 1;
+  width: 100vw;
+  height: 101%;
+  background: url(./image/slider/Slider.png) no-repeat center, #648bf0;
+}
+.element1 {
+  animation-name: pulse;
+  animation-duration: 5s;
+  animation-timing-function: ease-out;
+  animation-delay: 0s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+}
+
+@keyframes pulse2 {
+  0% {
+    transform: translateX(200vw);
+  }
+  50% {
+    transform: translateX(150vw);
+  }
+  100% {
+    transform: translateX(100vw);
+  }
+}
+.element2 {
+  animation-name: pulse2;
+  animation-duration: 5s;
+  animation-timing-function: ease-out;
+  animation-delay: 0s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-play-state: running;
+}
+@keyframes pulse3 {
+  0% {
+    transform: translateX(100vw);
+  }
+  50% {
+    transform: translateX(150vw);
+  }
+  100% {
+    transform: translateX(200vw);
+  }
+}
+.element3 {
+  animation-name: pulse3;
+  animation-duration: 5s;
+  animation-timing-function: ease-out;
+  animation-delay: 0s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-play-state: running;
+}
+@keyframes pulse4 {
+  0% {
+    transform: translateX(100vw);
+  }
+  50% {
+    transform: translateX(50vw);
+  }
+  100% {
+    transform: translateX(0vw);
+  }
+}
+.element4 {
+  animation-name: pulse4;
+  animation-duration: 5s;
+  animation-timing-function: ease-out;
+  animation-delay: 0s;
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-play-state: running;
+}
+
+.display {
+  display: none;
+}
+
+html {
+  font-size: 10px;
+  scroll-behavior: smooth;
+}
+* {
+  margin: 0;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.hamburger-wrapper {
+  display: none;
+}
+.rotate {
+  transform: rotate(90deg);
+}
+.hamburger {
+  width: 25px;
+  height: 20px;
+  position: relative;
+}
+.hamburger_line {
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 2.4px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  top: calc(50% - 1.5px);
+}
+.hamburger::before {
+  content: "";
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 3px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  top: 0;
+}
+.hamburger::after {
+  content: "";
+  display: inline-block;
+  background: var(--wite);
+  border-radius: 5px;
+  height: 3px;
+  width: 100%;
+  position: absolute;
+  left: 0px;
+  bottom: 0px;
+}
+.h2 {
+  font-family: "Lato900";
```

It is a good practice to specify fallback font's
```suggestion
  font-family: "Lato900" , sans-serif;
```